packages/@rnw-scripts/doxysaurus/src/renderer.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ **/
+
+//
+// Renders Markdown based documentation model to markdown files by applying Mustache templates.
+//
+
+import mustache from 'mustache';
+import path from 'path';
+import {log} from './logger';
+import {promises as fs} from 'fs';
+import {Config} from './config';
+import {DocModel} from './doc-model';
+
+const templateCache: {[index: string]: string} = {};
+
+export async function renderDocFiles(docModel: DocModel, config: Config) {
+  const renderedFiles: string[] = [];
+  const outputPath = path.join(config.output, 'out');
+  await fs.mkdir(outputPath).catch(err => {
```

Comment:
You could instead pass `{recursive: true}` to `mkdir` which will avoid errors on already exists, along with automatically creating any needed parent directories.

Reply:
Good suggestion! Fixed.